contrib/kind.sh

```diff
@@ -212,6 +212,10 @@ ovn_apiServerAddress=${API_IP} \
   ovn_num_worker=${KIND_NUM_WORKER} \
   j2 ${KIND_CONFIG} -o ${KIND_CONFIG_LCL}
 
+# use memory storage for etcd
```

Comment:
kind mounts /tmp inside the nodes in a tmpfs automatically, 
no need to create it and mount it

Reply:
this can be simplified to https://github.com/ovn-org/ovn-kubernetes/pull/1426

Reply:
@aojea thanks. Closing  this PR in favor of #1426